expo-maps/src/Maps.types.ts

```diff
@@ -15,4 +19,8 @@ export type ExpoMapViewProps = ViewProps &
   PropsWithChildren<{
     provider?: 'google' | 'apple';
     mapType?: 'normal' | 'hybrid' | 'satellite' | 'terrain';
+    enableRotateGestures: boolean;
+    enableScrollGestures: boolean;
+    enableTiltGestures: boolean;
+    enableZoomGestures: boolean;
```

Comment:
I think we can make these props optional since we use `defaultNativeExpoMapViewProps`, so user isn’t forced to specify them while using our library.

```suggestion
    enableRotateGestures?: boolean;
    enableScrollGestures?: boolean;
    enableTiltGestures?: boolean;
    enableZoomGestures?: boolean;
```

Reply:
Sure, arriving in next commit.